### ISSUE TITLE
mvmt2dt returns an error

### ISSUE THREAD

**Issue by raorben** [dbspitz/migrateR#2]
The codes were working previously so I think it could be an upgrade in a function somewhere, but I am not sure. I did reinstall migrateR. Often my birds don't stay long at their migration "destination" so this is true (jdts[2] > jdts[3]). When I remove that warning from the function I still get the first. 

mvmt2dt returns this error:
Error in t(gd) %*% vcov. : 
  requires numeric/complex matrix/vector arguments
 Show Traceback
 
 Rerun with Debug
 Error in if (jdts[2] > jdts[3]) { : missing value where TRUE/FALSE needed 

Code:
Migdate<-data.frame()
birds<-attributes(rnsd3)$names
for (i in 1:length(nt.rnsd1)){
  mod=nt.rnsd1[i]
  bird<-birds[i]
  Ti<-mvmt2dt(rnsd3[[i]],mod=mod,p=0.1)#returns the time when 
  #movements are 10% initiated or 90% complete
  (Ti <- do.call("c", Ti))

  TT<-data.frame(bird=bird,i=i,model=mod,dday1=Ti$dday[1],dday2=Ti$dday[2],dday3=Ti$dday[3],dday4=Ti$dday[4],
                date1=Ti$date[1],date2=Ti$date[2],date3=Ti$date[3],date4=Ti$date[4])
  Migdate<-rbind(Migdate,TT)
}

**Comment by raorben** [comment]
I just tried to install and build the vignettes and I get a similar error (see below)
library(devtools)
install_github("dbspitz/migrateR/migrateR",force = TRUE, build_vignettes = T)

I updated devtools and that didn't help.  I am running R version 3.3.2 (2016-10-31) -- "Sincere Pumpkin Patch" and the regular package loading seems to work just fine. 

ERROR:
* checking for file ‘/private/var/folders/3h/5x4j24ts76j3t5hfby90ww6h0000gn/T/RtmpBUXyim/devtools547c22c087a8/dbspitz-migrateR-58eaa60/migrateR/DESCRIPTION’ ... OK
* preparing ‘migrateR’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
* creating vignettes ... ERROR
Warning: convergence problem(s) for "s110 2008"
	resident  :		false convergence (8)

Warning: convergence problem(s) for "s110 2010"
	resident  :		false convergence (8)

Error in t(gd) %*% vcov. : 
  requires numeric/complex matrix/vector arguments
Error in t(gd) %*% vcov. : 
  requires numeric/complex matrix/vector arguments
Error in t(gd) %*% vcov. : 
  requires numeric/complex matrix/vector arguments
Error in t(gd) %*% vcov. : 
  requires numeric/complex matrix/vector arguments
Quitting from lines 491-493 (migrateR.Rmd) 
Error: processing vignette 'migrateR.Rmd' failed with diagnostics:
requires numeric/complex matrix/vector arguments
Execution halted
